flink-formats/flink-parquet/src/test/java/org/apache/flink/formats/parquet/protobuf/ParquetProtoStreamingFileSinkITCase.java

```diff
@@ -49,13 +50,12 @@
  * Simple integration test case for writing bulk encoded files with the {@link StreamingFileSink}
  * with Parquet.
  */
-public class ParquetProtoStreamingFileSinkITCase extends AbstractTestBase {
-
-    @Rule public final Timeout timeoutPerTest = Timeout.seconds(20);
+@ExtendWith(MiniClusterExtension.class)
+@Timeout(20)
```

Comment:
There was a community discussion on trying to avoid timeouts (see [discussion thread](https://lists.apache.org/thread/q11t0y3qsw07lkb811037h4mvx0r7ncn) and [corresponding Flink coding guidelines](https://flink.apache.org/contributing/code-style-and-quality-common.html#avoid-timeouts-in-junit-tests)).
The decision was only made on unit tests but I don't see why the same wouldn't also apply for integration tests. WDYT?